src/NodeCompiler/CompileNodeToValue.php

```diff
@@ -52,7 +50,7 @@ public function __invoke(Node $node, CompilerContext $context)
                 return $this->compileClassConstant($context);
             }
 
-            throw new Exception\UnableToCompileNode('Unable to compile expression: ' . get_class($node));
```

Comment:
This is dreadful and you should feel bad.

Reply:
IIRC that's your code...